lazer/contracts/solana/package.json

```diff
@@ -5,7 +5,6 @@
   "scripts": {
     "fix:format": "prettier --write **/*.*",
     "test:format": "prettier --check **/*.*",
-    "test:anchor": "CARGO_TARGET_DIR=\"$PWD/target\" anchor test",
```

Comment:
You can leave this in if you want, as long as you remove it from the turbo config it won't get triggered